benchmark/profile_generation.py

```diff
@@ -186,7 +186,7 @@ def profile_throughput(model_path: str, concurrency: int, input_seqlen: int,
     total_token_throughput = np.round(
         concurrency * test_round * (input_seqlen + output_seqlen) /
         elapsed_time, 2)
-    print(f'\n{"-" * 50}\ntotal time: {elapsed_time:.2f}s\n'
+    print(f'\n{'-' * 50}\ntotal time: {elapsed_time:.2f}s\n'
```

Comment:
no need to change this str format in `benchmark`

Reply:
> no need to change this str format in `benchmark`

it change after i run "pre-commit run --all-files",lvhan028 in wechat group  said it could solve lint failed; now is there anything else i can do?

Reply:
hi, you can pre-commit the only files  you changed. 
```
pip install pre-commit
pre-commit install
pre-commit run --file xx.py 
```

Reply:
> hi, you can pre-commit the only files you changed.
> 
> ```
> pip install pre-commit
> pre-commit install
> pre-commit run --file xx.py 
> ```

i have roll back and add compose.yml in internvl docs